src/month.rs

```diff
@@ -1,12 +1,11 @@
 //! The `Month` enum and its associated `impl`s.
 
+use core::convert::TryFrom;
 use core::fmt;
-#[cfg(feature = "parsing")]
 use core::num::NonZeroU8;
 
 use self::Month::*;
-#[cfg(feature = "parsing")]
-use crate::error;
+use crate::error::ComponentRange;
```

Comment:
Let's keep the `use crate::error` import and refer to it as `error::ComponentRange`. Without this, it's not immediately clear that the type is an error type.